src/table.rs

```diff
@@ -41,6 +42,26 @@ pub struct Table {
 }
 
 impl Table {
+    /// Creates a new table from a schema and a vector of record batches.
+    ///
+    /// Returns an error if a record batch's schema fields do not match the
+    /// top-level schema's fields.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use arrow_array::RecordBatch;
+    /// use arrow_schema::{Schema, SchemaRef};
+    /// use geoarrow::{GeometryArrayTrait, array::PointArray, table::Table};
+    ///
+    /// let point = geo::point!(x: 1., y: 2.);
```

Comment:
```suggestion
    /// let point = geo::point!{x: 1., y: 2.};
```
It looked like this needs to be square brackets? https://docs.rs/geo/latest/geo/macro.point.html
Does it work as is?

Reply:
If it works we don't need to switch it; I just didn't know it would work like that

Reply:
Yup, just checked in a test crate and the braces don't matter (you can use `[]`, `{}`, or `()` for macros):

```rust
fn main() {
    dbg!(geo::point!(x: 1., y: 2.));
    dbg!(geo::point! {x: 1., y: 2.});
}
```

Output:

```text
[src/main.rs:2:5] geo::point!(x: 1., y: 2.) = Point(
    Coord {
        x: 1.0,
        y: 2.0,
    },
)
[src/main.rs:3:5] geo::point! { x: 1., y: 2. } = Point(
    Coord {
        x: 1.0,
        y: 2.0,
    },
)
``